src/components/rules-creation-page/steps/IncludeExcludeSelectionStep.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useState, useEffect } from "react";
+import { withSnackbar } from "d2-ui-components";
+import SyncRule from "../../../models/syncRule";
+import i18n from "../../../locales";
+import { Toggle } from "../../toogle/Toogle";
+import { getMetadata } from "../../../utils/synchronization";
+import { getBaseUrl } from "../../../utils/d2";
+import { D2, ModelDefinition } from "../../../types/d2";
+import _ from "lodash";
+import Dropdown from "../../dropdown/Dropdown";
+import { d2ModelFactory } from "../../../models/d2ModelFactory";
+import { D2Model } from "../../../models/d2Model";
+import { MultiSelector } from "d2-ui-components";
+
+interface IncludeExcludeSelectionStepProps {
+    classes: any;
+    d2: D2;
+    syncRule: SyncRule;
+    onChange: (syncRule: SyncRule) => void;
+}
+
+interface ModelSelectItem {
+    name: string;
+    id: string;
+}
+
+interface friendlyRulesNamesDiccionary {
+    [metadataType: string]: string;
+}
```

Comment:
Since there is no usage of ``friendlyRulesNamesDiccionary`` out of ``includeExcludeRulesFriendlyNames`` we don't need to create an interface. It could be simplified to:

```
const includeExcludeRulesFriendlyNames: {
    [metadataType: string]: string;
} = {
...
}
```

Also since it's not an overly complex type, maybe TS already correctly infers it.

Reply:
Done!